crates/router/src/configs/defaults.rs

```diff
@@ -2048,20 +2138,20 @@ impl Default for super::settings::RequiredFields {
                                                 }
                                             ),
                                             (
-                                                "billing.address.first_name".to_string(),
+                                                "payment_method_data.billing.address.first_name".to_string(),
                                                 RequiredFieldInfo {
-                                                    required_field: "billing.address.first_name".to_string(),
+                                                    required_field: "payment_method_data.billing.address.first_name".to_string(),
                                                     display_name: "card_holder_name".to_string(),
-                                                    field_type: enums::FieldType::UserBillingName,
+                                                    field_type: enums::FieldType::UserFullName,
                                                     value: None,
                                                 }
                                             ),
                                             (
-                                                "billing.address.last_name".to_string(),
+                                                "payment_method_data.billing.address.last_name".to_string(),
```

Comment:
here, we are using `payment_method_data` 's last name but for fields line1, city, we are using `billing.address.line1`, `billing.address.city`. Is there any reason we are using two different objects?

Reply:
I have changed the key to refer the value from `billing.address.first_name` and `last_name`